src/app/decks/Decks.js

```diff
@@ -75,10 +86,28 @@ class Decks extends Component {
 
   isSaved = id => this.state.savedDecks.includes(id);
 
+  handleSearch = () => {
+    this.setState({redirect: true});
+  }
+
+  onKeyStroke = (e) => {
+    if (e.key === 'Enter') {
+      this.handleSearch();
+    }
+  }
+
   render() {
     const { location } = this.props;
     const { decks, isLoading, isError, savedDecks, activeTab } = this.state;
 
+    if (this.state.redirect) {
+      for(var i = 0; i< this.state.collections.length; i++){
+        if(this.state.collections[i].name === this.state.searchString){
```

Comment:
I'm sure there is a better way to do this besides iterating over the array to find the associated id, but I couldn't find one. Any help here @nlaz ?

Reply:
Thanks! 👍 👍 👍